docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,215 @@
+
+For complete details on Stargate, see the [Stargate documentation](https://stargate.io/docs/stargate/1.0/quickstart/quickstart.html).
+
+## Access Cassandra using CQLSH
+
+If you're familiar with Cassandra, then you're familiar with CQLSH. You can download a full-featured CQLSH utility from Datastax and use that to interact with K8ssandra as if you were in a native Cassandra environment.
+
+To access K8ssandra using the stand alone CQLSH utility:
+
+1. Make sure you have [Python 2.7](https://www.python.org/download/releases/2.7/) installed on your system.
+
+1. Download CQLSH from the  [DataStax download site](https://downloads.datastax.com/#cqlsh) choosing the version for **DataStax Astra**.
+
+1. Connect to Cassandra replacing `<k8ssandra-username>` and `<k8ssandra-password>` with the values you retrieved in [Retrieve K8ssandra superuser credentials]({{< relref "/docs/getting-started#superuser" >}}):
+
+    ```bash
+    cqlsh -u <k8ssandra-username> -p <k8ssandra-password>
+    Connected to k8ssandra at 127.0.0.1:9042.
+    [cqlsh 6.8.0 | Cassandra 3.11.6 | CQL spec 3.4.4 | Native protocol v4]
+    Use HELP for help.
+    k8ssandra-superuser@cqlsh>
+   ```
+
+1. Populate a CQL file with the following data and save as `test-data.cql`:
+
+    ```sql
+    CREATE KEYSPACE k8ssandra_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+    USE k8ssandra_test;
+    CREATE TABLE users (email text primary key, name text, state text);
+    insert into users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+    insert into users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+    insert into users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+    insert into users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+    ```
+
+1. Import the test data into the Cassandra node using the CQL `SOURCE` command:
+
+    ```bash
+    SOURCE 'test-data.cql';
+    ```
+
+1. Query the data:
+
+    ```sql
+    cqlsh> SELECT * FROM k8ssandra_test.users;
+
+     email             | name          | state
+    -------------------+---------------+-------
+     alice@example.com |   Alice Smith |    TX
+       bob@example.com |     Bob Jones |    VA
+     david@example.com |    David Yang |    NV
+     carol@example.com | Carol Jackson |    CA
+
+    (4 rows)
+    ```
+
+For complete details on Cassandra, CQL and CQLSH, see the [Apache Cassandra](https://cassandra.apache.org/) web site.
+
+## Next
+
+* For detailed information on additional K8ssandra tasks, see [Tasks]({{< relref "docs/topics" >}}).
+* For a list of frequently asked questions, see the [FAQs]({{< relref "docs/faqs" >}}).
+* For detailed information on K8ssandra, see [Architecture]({{< relref "docs/architecture" >}}).
+* For information on the various K8ssandra Helm charts, see [Architecture]({{< relref "docs/reference" >}}).
```

Comment:
I don't think the current architecture section really talks about individual charts. Since the previous bullet already links to architecture, maybe delete.

Reply:
This is just a pointer to the architecture information such as it is. The next one points to the chart references.